api/dsym.md

```diff
@@ -0,0 +1,24 @@
+To upload your .dSYM files, POST to the the `api/1/dsym` endpoint with the following params:
+
+access_token
+:	your `post_server_item` access token
+
+version
+:	a string indicating the current code version.
+
+bundle_identifier
+:	your bundle identifier
+
+dsym
+:	path to your zipped dSYM file. See [here](https://raw.githubusercontent.com/rollbar/rollbar-ios/master/upload_dsym.py) for more info on this.
```

Comment:
As far as the API is concerned, this needs to be the file itself, not the path to the file. I'm not sure what the best way to word that is - @coryvirok maybe you have a suggestion?